fake_robot.py

```diff
@@ -1,10 +1,19 @@
 import time
 
+import gflags
+
 from actions.action import ActionException
 from contextlib import contextmanager
 from parts.load_cell import FakeLoadCellMonitor
 from robot import Robot
 
+FLAGS = gflags.FLAGS
+gflags.DEFINE_boolean("fake_sleep", True, "sleep() in the fake")
```

Comment:
This help comment didn't immediately make sense to me. How about 'sleep() for the fake robot'?